src/CMakeLists.txt

```diff
@@ -305,12 +305,16 @@ include_directories("${CMAKE_SOURCE_DIR}/src/dmclock/support/src")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/googletest/googletest/include")
 
 if(WITH_SEASTAR)
-  find_package(fmt 4.0.0)
-  if(NOT fmt_FOUND)
+  find_package(fmt 4.0.0 QUIET)
+  if(fmt_FOUND)
+    message(STATUS "Found fmt: ${fmt_DIR}")
```

Comment:
libfmt is actually detected in the CONFIG mode, which does not print the detect result. so let's do it by ourselves.